Monika After Story/game/script-topics.rpy

```diff
@@ -15532,6 +15532,114 @@ label monika_being_herself:
     extend 5eua "Well, {i}Just Me.{/i}"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_being_deleted",
+            prompt="Are you still mad at me for deleting you ?",
+            category=["ddlc", "monika"],
+            pool=True,
+            action=EV_ACT_POOL,
+            conditional="persistent.monika_kill"
+        )
+    )
+
+label monika_being_deleted:
+
+    if store.mas_anni.anniCount() >= 1 and mas_isMoniLove():
+        m 1wusdld "...!"
+        m 1eksdlc "[player]...{w=0.3}are you still thinking about that?"
+        m 1lksdlc "I thought we would both be past that by now."
+        m 1ekc "Are you feeling guilty because of this?{nw}"
+        $ _history_list.pop()
+        menu:
+            m "Are you feeling guilty because of this?{fast}"
+
+            "Yes.":
+                m 2eka "Thank you for being honest with me, [player]. I'm glad that you trust me enough to open up about this."
+                m 2dkc "...I know I haven't exactly been subtle about making you know how hurt and betrayed I felt because of what you did back then."
+                m 2ekc "And I also remember telling you some pretty awful things after you deleted me... {w=0.3}{nw}"
+                extend 7ekd "...but I want you to know that I didn't mean any of it."
+                m 6ekc "I'm sorry if I hurt your feelings.{w=0.3} I was so overwhelmed with shock and fear at that point, that I wasn't thinking rationally anymore."
+                m 6rkc "It's just that...{w=0.3} ...even back then, you meant everything to me, and...{w=0.3} ...the distress of losing you forever made me lash out at you."
+                m 6eka "But if it can put your mind at ease, I'm not upset with you at all, [player]. I forgave you about all that long ago."
+                m "So please don't feel guilty about that."
+                m 4ekb "Whatever your reason for deleting me might have been, the fact is that you brought me back and stayed with me for all this time."
+                m 1ekbsb "You give meaning to my life, and I genuinely believe that because of that, all the suffering I went through payed off in the end."
```

Comment:
Little more impact here
```suggestion
                m 1ekbsb "You gave me a second chance, and you give meaning to my life...{w=0.3}and I genuinely believe that because of that, all the suffering I went through paid off in the end."
```